recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -8,3 +8,10 @@ export YOCTO_ALTERNATE_EXE_PATH = "${STAGING_LIBDIR}/llvm-config"
 
 PACKAGECONFIG[gallium-llvm] = "-Dllvm=true -Dshared-llvm=true, -Dllvm=false, clang clang-native \
 ${@'elfutils' if ${GALLIUMDRIVERS_LLVM33_ENABLED} else ''}"
+
+do_write_config_append() {
+    # meson.bbclass sets llvm-config to llvm-config${LLVMVERSION}.  As we're
+    # dropping the version from the YOCTO_ALTERNATIVE_EXEC_PATH above, we also
+    # need to do so here.
+    sed -i "s|llvm-config[0-9.]*|llvm-config|g" ${WORKDIR}/meson.cross
+}
```

Comment:
with https://github.com/kraj/meta-clang/commit/459814ea4768bdab50efe7e4e918faf658ed6f42 we should not need this hunk.

Reply:
Ah cool, didn't see that in master and thought it had been left out intentionally.  I cherry-picked it (updating version to 9.0.0) into this PR and dropped this commit.

Reply:
@jsbronder currently zeus is in sync with master, so it would be better for us to merge zeus from master

Reply:
Not sure what you mean by "in sync".  Right now there's a number of differences.  Of relevance here is the addition of LLVMVERSION and master has 9.0.1 whereas zeus has 9.0.0.

https://github.com/kraj/meta-clang/compare/zeus...master

Anyways, let me know what, if anything, you want me to do in this PR.